packages/fxa-profile-server/lib/subscriptions.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This is an abbreviated version of the function from
+// fxa-auth-server/lib/routes/utils/subscriptions.js
+exports.determineClientVisibleSubscriptionCapabilities = function(
+  clientId,
+  allCapabilities
+) {
+  const capabilitiesToReveal = new Set([
+    ...(allCapabilities['*'] || []),
+    ...(allCapabilities[clientId] || []),
+  ]);
+  return capabilitiesToReveal.size > 0
+    ? Array.from(capabilitiesToReveal).sort()
+    : undefined;
```

Comment:
Is this going to be an issue?  Feels like it'd be safer to return an empty list.

Reply:
This is so we don't include the subscriptions property at all in profiles without subscriptions. We didn't want to add a new empty property to all profiles